platform/ios/src/MGLMapView.mm

```diff
@@ -770,6 +773,49 @@ - (void)layoutSubviews
     }
 }
 
+/// Updates `contentInset` to reflect the current window geometry.
+- (void)adjustContentInset
+{
```

Comment:
Does this handle the case where  MGLMapView is nested more than 2 level deep in the UIViewController view hierarchy?